plugins/communications-app/RecipientsForm/index.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Form } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { useSelector, useDispatch } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context';
+import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
+
+import './styles.scss';
+
+const disableIsHasLearners = ['track', 'cohort'];
+
+const RecipientsForm = (props) => {
+  const { cohorts: additionalCohorts } = props;
+  const formData = useSelector((state) => state.form);
+  const dispatch = useDispatch();
+  const { isEditMode, emailRecipients, isFormSubmitted } = formData;
+
```

Comment:
What are your thoughts about this:

```suggestion
const RecipientsForm = ({ cohorts: additionalCohorts }) => {
  const dispatch = useDispatch();
  const { isEditMode, emailRecipients, isFormSubmitted } = useSelector(({ form }) => form);
```